index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Password Generator</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&family=Karla:wght@800&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="app">
+        <div class="header">
+            <div>
+                <h1> 
+                    <span class="top"> Generate a </span> <br>
+                    <span class="bottom"> random password</span>
+                </h1>
+            </div>
+            <div>
+                <p>Never use an insecure password again.</p>
+            </div>
+        </div>
+        <div class="content">
+            <div class="btns">
+            <button id="btn" onclick="password()">Generate passwords</button>
+            <button id="newbtn" onclick="newpassword()">New password</button>
```

Comment:
It was addditional task for add some changes.

Reply:
Okay, good. But instead of button with same functionality, you can add different functionality such as different background colors for each random password.It can be good exercise as well 🙂